pypeit/specobjs.py

```diff
@@ -274,7 +279,13 @@ def get_std(self, multi_spec_det=None):
                 for iord in range(norders):
                     ind = (self.ECH_FRACPOS == uni_objid[iobj]) & (self.ECH_ORDER == uni_order[iord])
                     spec = self[ind]
-                    SNR[iord, iobj] = np.median(spec[0].OPT_COUNTS*np.sqrt(spec[0].OPT_COUNTS_IVAR))
+                    # Grab SNR
+                    if hasattr(spec[0], 'OPT_COUNTS'):
```

Comment:
Same comment above applies here.

Reply:
Also agreed. And modified.
And changed `self` to `spec[0]`